core/storage/classifier/gae_models_test.py

```diff
@@ -109,8 +109,7 @@ def test_query_new_and_pending_training_jobs(self):
         self.assertEqual(
             training_jobs[1].status,
             feconf.TRAINING_JOB_STATUS_PENDING)
-        self.assertEqual(more, False)
-        self.assertEqual(cursor is not None, True)
+        self.assertEqual(offset, 2)
```

Comment:
Do we also have a test to verify that a max of 10 are fetched at a time? That seems useful to verify (since it's part of ensuring the offset is an incrementing index).

Reply:
Yes I think it will be good if we test this. Done